src/TLuaInterpreter.cpp

```diff
@@ -5875,29 +5875,31 @@ int TLuaInterpreter::setButtonStyleSheet(lua_State* L)
     QString name, css;
 
     if (!lua_isstring(L, 1)) {
-        lua_pushstring(L, "setButtonStyleSheet: wrong first arg");
-        lua_error(L);
-        return 1;
+        lua_pushfstring(L, "setButtonStyleSheet: bad argument #1 type (string expected, got %s!)", luaL_typename(L, 1));
+        return lua_error(L);
     } else {
-        name = lua_tostring(L, 1);
+        name = QString::fromUtf8(lua_tostring(L, 1));
     }
     if (!lua_isstring(L, 2)) {
-        lua_pushstring(L, "setButtonStyleSheet: wrong second arg");
-        lua_error(L);
-        return 1;
+        lua_pushfstring(L, "setButtonStyleSheet: bad argument #2 type (string expected, got %s!)", luaL_typename(L, 2));
```

Comment:
`css as string`. The more informative, the better. While staying consise, there's a fine balance